samza-api/src/main/java/org/apache/samza/startpoint/Startpoint.java

```diff
@@ -22,11 +22,14 @@
 import com.google.common.base.Objects;
 import java.time.Instant;
 import org.apache.samza.annotation.InterfaceStability;
+import org.codehaus.jackson.annotate.JsonTypeInfo;
+
```

Comment:
nit: Minor. Please remove the empty line.

Reply:
Sure